src/pal/src/safecrt/internal_securecrt.h

```diff
@@ -23,6 +23,17 @@
 #ifndef _INC_INTERNAL_SECURECRT
 #define _INC_INTERNAL_SECURECRT
 
+#if defined(__linux__) && !defined(__GLIBC__)  // e.g. OS with musl-libc
```

Comment:
__GLIBC__ isn't defined on other platforms we support either (so these typedefs will also apply on, say, Ubuntu as well).